run.sh

```diff
@@ -17,5 +17,5 @@ __dotnet=$__toolsLocalPath/dotnetcli/dotnet
 
 cp -fR $__scriptpath/tools-override/* $__toolsLocalPath 
 
-$__dotnet $__toolsLocalPath/run.exe $*
-exit $?
\ No newline at end of file
+DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1 $__dotnet $__toolsLocalPath/run.exe $*
```

Comment:
I think this should also happen for Windows.

Also, why do we need this twice?  I saw it in build.proj as well?